library/ix-dev/charts/elastic-search/questions.yaml

```diff
@@ -0,0 +1,176 @@
+groups:
+  - name: Elastic Search Configuration
+    description: Configure Elastic Search
+  - name: User and Group Configuration
+    description: Configure User and Group for Elastic Search
+  - name: Network Configuration
+    description: Configure Network for Elastic Search
+  - name: Storage Configuration
+    description: Configure Storage for Elastic Search
+  - name: Resources Configuration
+    description: Configure Resources for Elastic Search
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+
+  - variable: esConfig
+    label: ""
+    group: Elastic Search Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: password
+          label: Password
+          description: The password for the Elastic Search (elastic) user.
+          schema:
+            type: string
+            default: ""
+            required: true
+        - variable: nodeName
+          label: Node Name
+          description: The name of the Elastic Search node.
+          schema:
+            type: string
+            default: elastic_search_node
```

Comment:
Probably better to quote and below as well just for consistency for string schema?
Might bite us in future with something being considered as boolean/integer and not string or something..